modules/apps/layout/layout-page-template-api/src/main/resources/com/liferay/layout/page/template/validator/dependencies/page_definition_json_schema.json

```diff
@@ -300,25 +376,23 @@
 				"mapping": {
 					"additionalProperties": false,
 					"properties": {
-						"collectionItemFieldKey": {
-							"title": "The collectionItemFieldKey Schema",
-							"type": "string"
-						},
 						"fieldKey": {
 							"title": "The fieldKey Schema",
 							"type": "string"
 						},
-						"itemClassName": {
-							"title": "The itemClassName Schema",
-							"type": "string"
-						},
-						"itemClassPK": {
-							"title": "The itemClassPK Schema",
-							"type": "integer"
+						"itemReference": {
```

Comment:
I think this does a good job at expressing the real semantics of the relationship :)